sources/platform/actors/publishing/monetize.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Monetize your actor
+description: TODO
+slug: /actors/publishing/monetize
```

Comment:
In "docs/platform/actors/publishing" section I centralized our content for public+paid actor devs into these 3 pages:

![obrazek](https://user-images.githubusercontent.com/594801/223745679-ef86557d-a3df-4e5e-ae74-e8cb41ee2a6d.png)

There is some duplicate content with academy, we need to decide how to approach this. @mnmkng @PerVillalva